src/components/TextPill.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import Text from './Text';
+import styles from '../styles/styles';
+import stylePropTypes from '../styles/stylePropTypes';
+
+const propTypes = {
+    /** Text to show */
+    children: PropTypes.string,
+
+    /** Text additional style */
+    style: stylePropTypes,
+};
+
+const defaultProps = {
+    children: '',
```

Comment:
Instead of setting this empty, we should make the property mandatory

Reply:
Done.